src/SourcemapToolkit.SourcemapParser/SourceMapTransformer.cs

```diff
@@ -18,6 +18,7 @@ public static SourceMap Flatten(SourceMap sourceMap)
                 Version = sourceMap.Version,
                 Mappings = sourceMap.Mappings,
                 Sources = sourceMap.Sources == null ? null : new List<string>(sourceMap.Sources),
+                SourcesContent = sourceMap.SourcesContent == null ? null : new List<string>(sourceMap.SourcesContent),
```

Comment:
Can you add a simple test to cover the sources content population?

Reply:
+